contracts/token/ERC721/base/ERC721BaseInternal.sol

```diff
@@ -53,25 +57,26 @@ abstract contract ERC721BaseInternal is IERC721Internal {
         returns (bool)
     {
         require(
-            ERC721BaseStorage.exists(tokenId),
+            ERC721BaseStorage.layout().exists(tokenId),
```

Comment:
For consistency perhaps use the same approach as above in  `_getApproved` and below in `_mint`?

Reply:
In what sense?  Defining the `l` variable on a separate line?  Generally I only do that when it needs to be accessed multiple times.

Reply:
Oh ofcourse. My oversight, thanks for clarifying.